k2/csrc/fsa_equivalent.cc

```diff
@@ -7,14 +7,118 @@
 #include "k2/csrc/fsa_equivalent.h"
 
 #include <algorithm>
+#include <queue>
 #include <random>
 #include <unordered_map>
 #include <unordered_set>
+#include <utility>
 #include <vector>
 
 #include "k2/csrc/fsa.h"
 #include "k2/csrc/fsa_algo.h"
 #include "k2/csrc/properties.h"
+#include "k2/csrc/util.h"
+#include "k2/csrc/weights.h"
+
+namespace {
+// out_weights[i] = weights[arc_map1[arc_map2[i]]]
+void GetArcWeights(const float *weights, const std::vector<int32_t> &arc_map1,
```

Comment:
can we make things static if they are local to the file?
I know the anon namespace guarantees this, I just feel like it would be clearer.

Reply:
mm..OK, I just added it and remove just before..

Reply:
How about replacing unnamed namespace with static ?